### DISCUSSION TITLE
Additional Help

### DISCUSSION THREAD

**Discussion by harryajc** [netbox-community/devicetype-library#2330] [General]
Hi,

Just wondering as there are over 120 pending pull requests, if some additional moderators are wanted. I am a netbox users and would be happy to help.

Harry

**Comment by grbeneke** [comment]
Hi,

I would also be willing to assist as a maintainer. There is clearly a lot of interest in this project and I would not like to see contributors becoming disillusioned.

Graham

**Comment by DanSheps** [comment]
Thanks for the interest.  We will discuss it amoungst the maintainers and get back to you.

**Reply by DanSheps** [reply]
@harryajc and @grbeneke If you are on the netdev community slack please reach out to me.  This kind of dropped off our radar.

**Reply by danner26** [reply]
Yes please do reach out. I would be happy to help onboard anyone that is a good fit (with prior approval from the maintainer team of course).

**Reply by danner26** [reply]
@harryajc welcome! Shoot me a message on Slack so we can collaborate. Happy to have you here!
